assets/js/package_search.js

```diff
@@ -6,16 +6,22 @@
         fortranLang.loadJSON(fortranLang.baseurl+'/packages/package_index.json', search);
 
     }
+    
 
     
     function search(data){
         // Called after json data is loaded
-        //
         
         projects = data.projects;
-
+	    
+        // In the earlier code there has been an error in the regex implemented in  function fortranLang.findGetParameter('query')
+	// Opened a new Issue for this  #388   
+	    
+        var url = new URL(window.location.href); //converts the url from index page and extracts the parameter "query" 
+       
         // Get search string
-        var queryString = fortranLang.findGetParameter('query').replace(/\+/g," ").replace(/"/g,'');
```

Comment:
For completeness, could you also delete the old `findGetParameter` function from [`fortranlang.js`](https://github.com/fortran-lang/fortran-lang.org/blob/master/assets/js/fortranlang.js) since it is now unused.

Reply:
Done @LKedward  :+1: